src/native/tools/malloc_napi.cpp

```diff
@@ -0,0 +1,25 @@
+/* Copyright (C) 2016 NooBaa */
+#include "../util/napi.h"
+
+#include <stdlib.h>
+#include <jemalloc/jemalloc.h>
```

Comment:
Where is this header coming from? I think we want to have this as an optional thing because it will fail on local builds

Reply:
currently is seems to fail on none-local builds as well :) 
how will we set it optional? I was thinking maybe to add jemalloc to the builder docker...